docs/update-versions.py

```diff
@@ -23,11 +23,12 @@ def main(gh_pages_dir):
     gh_pages_dir = Path(gh_pages_dir)
 
     all_versions = set()
-    stable_version = parse("")
+    stable_version = parse("0.0")
```

Comment:
The reason this was parsing an empty string was to get a version instance with no value, such that it could be used in comparisons with other versions, but would also be falsy. Setting this to the parsed value of `"0.0"` will result in a version instance that would be truthy.

I think it would be best to do something like this so we can retain the default for comparison below:
```suggestion
    default_version = parse("0.0")
    stable_version = stable_version
```

Reply:
Good poijnt @jkeifer, I like the suggestion. Fixed in new commit.